src/Security/Handler/SecurityHandlerInterface.php

```diff
@@ -23,22 +23,18 @@ interface SecurityHandlerInterface
     /**
      * @param string|array $attributes
      * @param mixed|null   $object
-     *
-     * @return bool
      */
-    public function isGranted(AdminInterface $admin, $attributes, $object = null);
+    public function isGranted(AdminInterface $admin, $attributes, $object = null): bool;
```

Comment:
The only call what we're doing are in
```
SonataAdminExtension::isGrantedAffirmative
```
and
```
AbstractAdmin::isGranted
```

In both case, we're passing `?object`.

So I think that we can restrict `$object` type to `?object` here.

Same for implementations.